modules/caddytls/connpolicy.go

```diff
@@ -80,6 +80,7 @@ func (cp ConnectionPolicies) TLSConfig(ctx caddy.Context) *tls.Config {
 	}
 
 	return &tls.Config{
+		MinVersion: tls.VersionTLS12,
```

Comment:
Why add this here?

Reply:
This is `G402: TLS MinVersion too low. (gosec)`